geode/src/main/java/com/yahoo/ycsb/db/GeodeClient.java

```diff
@@ -64,23 +64,23 @@
  * @author Swapnil Bawaskar (sbawaska at vmware)
```

Comment:
while you're here, would you mind clearing out @author tags? we rely on git commit metadata for authorship attribution.